docs/ldml/tr35-keyboards.md

```diff
@@ -1605,21 +1724,20 @@ _Attribute:_ `from` (required)
 
 - **Fixed character classes and escapes**
 
-    `\s \S \t \r \n \f \v \\ \$`
+    `\s \S \t \r \n \f \v \\ \$ \d \w \D \W` … TODO: check these
 
     The value of these classes do not change with Unicode versions.
 
     `\s` for example is exactly `[\f\n\r\t\v\u00a0\u1680\u2000-\u200a\u2028\u2029\u202f\u205f\u3000\ufeff]`
 
     `\\` and `\$` evaluate to `\` and `$`, respectively.
 
-- **Ranges and UnicodeSets**
+- **Character classes**
 
-    `[abc]` `[^abc]` `[a-z]` `[ॲ ऄ-आ इ-ऋ]` `[\u093F-\u0944 \u0962 \u0963]`
+    `[abc]` `[^def]`
```

Comment:
```suggestion
    `[abc]` `[^def]` `[a-z]`
```

Reply:
Should say also that we could represent the other two examples just by deleting the whitespace in them
```suggestion
    `[abc]` `[^def]` `[a-z]` `[ॲऄ-आइ-ऋ]` `[\u093F-\u0944\u0962\u0963]`
```